config/database.yml

```diff
@@ -1,8 +1,14 @@
+# If you want to change this file, please keep the changes in your working
+# copy by using
+#
+#     git update-index --skip-worktree config/database.yml
```

Comment:
can we add a suggestion to use environment variables and as a last option the update-index thing?